src/daos-serialize/daos-serialize.c

```diff
@@ -0,0 +1,226 @@
+#include <stdio.h>
+#include <getopt.h>
+#include <string.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <inttypes.h>
+#include <unistd.h>
+#include <errno.h>
+#include <stdbool.h>
+
+/* for daos */
+#ifdef DAOS_SUPPORT
+#include "mfu_daos.h"
+#endif
+#include "mpi.h"
+#include "mfu.h"
+
+#include "mfu_errors.h"
+
+/** Print a usage message. */
+void print_usage(void)
+{
+    printf("\n");
+    printf("Usage: daos-serialize [options] /<pool>/<cont> /<pool>/<cont> ... \n");
+    printf("\n");
+    printf("DAOS paths can be specified as:\n");
+    printf("       /<pool>/<cont> | <UNS path>\n");
+    printf("\n");
+    printf("Options:\n");
+    printf("  -v, --verbose            - verbose output\n");
+    printf("  -h, --help               - print usage\n");
+    printf("For more information see https://mpifileutils.readthedocs.io.\n");
+    printf("\n");
+    fflush(stdout);
+}
+
+int main(int argc, char** argv)
+{
+    /* assume we'll exit with success */
+    int rc = 0;
+
+    /* initialize MPI */
+    MPI_Init(&argc, &argv);
+    mfu_init();
+
+    /* get our rank */
+    int rank;
+    MPI_Comm_rank(MPI_COMM_WORLD, &rank);
+
+    /* verbose by default */
+    mfu_debug_level = MFU_LOG_VERBOSE;
```

Comment:
@dsikich Since verbose is turned on by default, maybe we should also add a quite option:
```
case 'q':
                mfu_debug_level = MFU_LOG_NONE;
```